core/arch/arm/plat-zynqmp/platform_config.h

```diff
@@ -56,8 +59,9 @@
 #define IT_CONSOLE_UART		IT_UART0
 #define CONSOLE_UART_CLK_IN_HZ	UART0_CLK_IN_HZ
 
-#define DRAM0_BASE		0
-#define DRAM0_SIZE		0x80000000
+#if !defined(CFG_DDR_SIZE)
+#define CFG_DDR_SIZE		0x80000000
```

Comment:
How about assigning the default value in `core/arch/arm/plat-zynqmp/conf.mk` instead?

Reply:
I believe like in imx it is done like that. I am OK for that.

Reply:
Added commit for that. Please check it out.